Pages/image.html

```diff
@@ -379,12 +379,12 @@
 </div>
 
 <div class="gallery">
-    <img src="../Content/images/photos_age/young%20age.jpg" alt="Merwan Sheriar Irani" width="300" height="200">
+   <img src="../Content/images/photos_age/young_age.jpg" alt="Merwan Sheriar Irani" width="130" height="253">
     <div class="desc">Baba at young age</div>
 </div>
 
 <div class="gallery">
-    <img src="../Content/images/photos_age/avatar.jpg" alt="Avatar Meher Baba" width="300" height="200">
+    <img src="../Content/images/photos_age/avatar.jpg" alt="Avatar Meher Baba" width="170" height="200">
     <div class="desc">Baba after Enlightened</div>
```

Comment:
The caption of the third picture not visible on the webpage.

Reply:
worked on it, now everything seems fine.